NycBasics/models.py

```diff
@@ -44,12 +94,29 @@ def __str__(self):
             + "_lat_"
             + str(self.wifi_latitude)
         )
+        """
+        return (
+            self.name
+            + "_lng_"
+            + str(self.wifi_Longitude)
+            + "_lat_"
+            + str(self.wifi_Latitude)
+        )
+        """
 
 
 class parking_model(models.Model):
     name = "parking"
     parking_latitude = models.FloatField(null=True)
     parking_longitude = models.FloatField(null=True)
+    """
+    parking_the_geom = models.CharField(max_length = 10000, null=True)
```

Comment:
Can the maximum size for the_geom be increased further? One of the multipolygon points has string length over 10000 characters.